closed/OpenJ9.gmk

```diff
@@ -471,16 +496,11 @@ run-preprocessors-j9 : stage-j9 \
 			SPEC=$(OPENJ9_BUILDSPEC) \
 			UMA_OPTIONS_EXTRA="-buildDate $(shell date +'%Y%m%d')" \
 			tools
-
+endif
```

Comment:
Please add a comment clarifying what is ending (and restore the blank line it replaced).